signatory.yaml

```diff
@@ -6,35 +6,15 @@ server:
 
 vaults:
   # Name is used to identify backend during import process
-  kms:
-    driver: cloudkms
+  local_file_keys:
```

Comment:
Please restore the original example configuration file.

Reply:
I restored the original one.